### PR TITLE
ParseEmailFiles - fails on timeout when body is very long

### PR DESCRIPTION
**Pull request by tallieber**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [#39063](https://github.com/demisto/etc/issues/39063)

## Description
When searching for mime encoded words using regex, long message body fails on timeout. encoded-words appear on message headers only.  Could not add a UT, the original message that fails contains a lot of secrets and it does not reproduce with just a very long text.

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by tallieber** [comment]
@Itay4 If you can just have a look, because it changes the code you recently added

File: Packs/CommonScripts/ReleaseNotes/1_4_7.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### ParseEmailFiles
+- Fixed an issue where the script gets timeout error with long message body as input.
```
Comment (bakatzir) [inline Packs/CommonScripts/ReleaseNotes/1_4_7.md:4]:
```suggestion
- Fixed an issue where the script fails on timeout intermittently when receiving long emails.
```

**Comment by Itay4** [comment]
can we add test?

**Review by yuvalbenshalom — APPROVED** [review]

**Comment by tallieber** [comment]
> can we add test?

@Itay4 just the original message fails, any generated message did not fail. I will ask the customer if we can use this message as test

**Pull request merged**